src/AsmResolver.DotNet/Cloning/MemberCloneContext.cs

```diff
@@ -12,10 +12,18 @@ public class MemberCloneContext
         /// Creates a new instance of the <see cref="MemberCloneContext"/> class.
         /// </summary>
         /// <param name="module">The target module to copy the cloned members into.</param>
-        public MemberCloneContext(ModuleDefinition module)
+        public MemberCloneContext(ModuleDefinition module) : this(module, null) { }
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="MemberCloneContext"/> class.
+        /// </summary>
+        /// <param name="module">The target module to copy the cloned members into.</param>
+        /// <param name="importerInstantiator">The instantiator for creating the reference importer</param>
+        public MemberCloneContext(ModuleDefinition module,
+            Func<MemberCloneContext, CloneContextAwareReferenceImporter>? importerInstantiator)
```

Comment:
If we only accept `CloneContextAwareReferenceImporter` instances here, do you think we should change the type of `MemberCloneContext.Importer`, for consistency?

Reply:
That would be best for consistency, though I think it is a binary breaking change.

Reply:
True, let's leave it as it is then for now.